sdk/state/close.go

```diff
@@ -122,9 +124,8 @@ func (c *Channel) ConfirmCoordinatedClose(ca CloseAgreement) (closeAgreement Clo
 	authorized = true
 	c.latestAuthorizedCloseAgreement = CloseAgreement{
 		Details:               ca.Details,
-		CloseSignatures:       appendNewSignatures(c.latestUnauthorizedCloseAgreement.CloseSignatures, ca.CloseSignatures),
-		DeclarationSignatures: c.latestUnauthorizedCloseAgreement.DeclarationSignatures,
+		CloseSignatures:       ca.CloseSignatures,
+		DeclarationSignatures: c.latestAuthorizedCloseAgreement.DeclarationSignatures,
 	}
-	c.latestUnauthorizedCloseAgreement = CloseAgreement{}
```

Comment:
I don't know why I removed this, it should stay, or rather there's little benefit to removing it but it is cleaner for the proposers side.